app/src/main/res/values/styles.xml

```diff
@@ -28,7 +28,7 @@
         <item name="android:paddingStart">@dimen/edit_text_margin</item>
         <item name="android:maxLines">1</item>
         <item name="android:drawablePadding">@dimen/edit_text_drawable_padding</item>
-        <item name="android:fontFamily">@fonts/font_fontFamily_widechat</item>
+        <item name="android:fontFamily">@string/font_fontFamily_widechat</item>
```

Comment:
I guess the same approach question as above here re: whether we're separating out these kinds of changes or not from non-widechat

Reply:
Same response......  xml files are tricky because we cannot use conditionals to my knowledge and non-widechat is not a top priority when it comes to a few cosmetic changes?